src/components/Strings.js

```diff
@@ -1,19 +1,19 @@
 export function NSamples({ total }) {
   if (!total) {
     return 'No Samples';
-  } else if (total === 1) {
+  }
+  if (total === 1) {
     return '1 Sample';
-  } else {
-    return `${total} Samples`;
   }
+  return `${total} Samples`;
 }
 
 export function NDownloadableSamples({ total }) {
   if (!total) {
     return 'No Downloadable Samples';
-  } else if (total === 1) {
+  }
+  if (total === 1) {
```

Comment:
Same question: could `total` be zero?

Reply:
same, for that case this should return 'No Downloadable Samples'

Reply:
:+1: